src/hooks/useCheckRevision.ts

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 
+import useAlert from './useAlert';
 import useFocusInput from './useFocusInput';
 
 const useCheckRevision = () => {
+  const { useSetAlert } = useAlert();
   const { handleChildClick } = useFocusInput();
   const [checked, setChecked] = React.useState<string[]>([]);
```

Comment:
optional: I wonder if using a `Set<string>` could be a better fit, seeing the operations you're doing here.

Reply:
I'll leave this as is for now, and file an issue to consider using Sets instead. it seems like it's worthwhile when handling many items, but this array will never have more than 4 elements. at least for while :) #137

Reply:
Yeah, to be clear, this wasn't a performance-oriented reason, more a simplification of the code, where you wouldn't need to check for existence before pushing :-)